src/components/Loading/index.js

```diff
@@ -0,0 +1,3 @@
+import Loading from './Loading';
```

Comment:
You can go with `export { default } from './Loading'.`

This is general comment for the whole project.